phpunit.xml.dist

```diff
@@ -12,23 +12,26 @@
          bootstrap="Tests/bootstrap.php"
 >
     <testsuites>
-        <testsuite name="User Bundle Test Suite">
-            <directory>./Tests/</directory>
+        <testsuite name="UserBundle Test Suite">
```

Comment:
I will change all phpunit config files. Because none of them include word Sonata. I agree with you on the namings